QuickOSM/test/test_overpass_panel.py

```diff
@@ -13,40 +17,68 @@
 
 class TestQuickOSMWidget(unittest.TestCase):
 
+    def setUp(self):
+        self.dialog = Dialog(get_iface())
+        layer_path = plugin_test_data_path('layer', 'layertest.qpkg')
+        self.layer = QgsVectorLayer(layer_path, 'layer', 'memory')
+        QgsProject.instance().addMapLayer(self.layer)
+        index = self.dialog.combo_query_type_qq.findData('layer')
+        self.dialog.combo_query_type_qq.setCurrentIndex(index)
+        self.dialog.selection_features[Panels.QuickQuery].setChecked(True)
```

Comment:
But I think you can move these lines in the test itself. We don't need for nomimatim for instance.

Only `self.dialog` maybe